### DISCUSSION TITLE
[VOTING POOL] Facing lag

### DISCUSSION THREAD

**Discussion by poteitouweb3** [sunflower-land/sunflower-land#2423] [Polls]
Since the node balacing + landscaping preview I have noticed a lot of lag on mobile and I have noticed some players reporting lag on mobile but also on PC. I am starting a voting pool for the community to help measure the problem.

**Comment by future2back** [comment]
Could you provide further details about the issue that was reported? Thank you.

**Reply by NomiceBinocle** [reply]
on mobile at certain times the game lags (resource harvesting has a delay between the actual click and the display) also the display freezes. I will try to take a video next time (I'm on iphone 12 pro to clarify, it's not a power problem)

**Comment by poteitouweb3** [comment]
Apparently after the approval of PR #2417 the lags have been largely resolved, so I am closing the pool and we can vote for a new one if something comes up. Thanks to everyone who voted!
